### ISSUE TITLE
Problem with version 8eeea0e

### ISSUE THREAD

**Issue by TheRealBadRobot** [thomasloven/lovelace-layout-card#36]
Hello, I installed the version 8eeea0e via HACS but there is no any card in panel mode. The releases page is empty too.

**Comment by rafuz** [comment]
Same here, just updated and the pages are empty in panel mode.

**Comment by VDRainer** [comment]
Error in browser console
```
TypeError: this.isPanel is not a function113 layout-card.js:5:14
```
The 113 in the error counts up 2-3 times a second.

**Comment by thomasloven** [comment]
Should be fixed in version 2.0
